eZ/Publish/API/Repository/Tests/Values/User/Limitation/LanguageLimitationTest.php

```diff
@@ -490,6 +493,338 @@ public function testPublishVersionTranslationWhenUserHasNoAccessToAllLanguages()
         $contentService->publishVersion($draft->versionInfo, [self::GER_DE, self::ENG_GB]);
     }
 
+    /**
+     * @dataProvider providerForPrepareDataForTestsWithLanguageLimitationAndDifferentContentTranslations
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    public function testCopyContentWithLanguageLimitationAndDifferentContentTranslations(
+        array $limitationValues,
+        bool $containsAllTranslations
+    ): void {
+        $repository = $this->getRepository();
+        $contentService = $repository->getContentService();
+
+        $content = $this->testPrepareDataForTestsWithLanguageLimitationAndDifferentContentTranslations(
+            $limitationValues,
+            $containsAllTranslations
+        );
+
+        $locationCreateStruct = new LocationCreateStruct(['parentLocationId' => 2]);
+
+        if ($containsAllTranslations) {
+            $clonedContent = $contentService->copyContent($content->contentInfo, $locationCreateStruct);
+
+            self::assertSame($content->getVersionInfo()->languageCodes, $clonedContent->getVersionInfo()->languageCodes);
+        } else {
+            self::expectException(UnauthorizedException::class);
```

Comment:
This method is not static (despite not actually using any object properties).
```suggestion
            $this->expectException(UnauthorizedException::class);
```